cmd/kubeadm/app/cmd/config.go

```diff
@@ -502,13 +504,30 @@ type ImagesList struct {
 	cfg *kubeadmapi.InitConfiguration
 }
 
+// ConfigImagesList represents information for JSON and YAML output produced by 'kubeadm config images list'
+type ConfigImagesList struct {
+	Images []string
```

Comment:
the "bike-shed" topic here is if we want to use `json` tags and camelCase the output fields or not.

version:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/version.go#L58
doesn't do that.

Reply:
This code already uses json field tags to convert to camelCase: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1beta1/types.go#L226-L246

How can we output bootstrap tokens without conversion then?

Reply:
actually my mistake:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/version.go#L36
`kubeadm version` does use `json` tags.

so i guess we should do camelCase everywhere based on the existing machine readable way...
i.e. we need a tag here for Images (`images`) in ConfigImagesList.